canary-checker/docs/scripting/cel.md

```diff
@@ -1372,7 +1365,7 @@ regexp.QuoteMeta("[a-z].*"); // "\\[a\\-z\\]\\.\\*"
 Syntax:
 
 ```javascript
-regexp.Replace(pattern, replacement, input)
+regexp.Replace(pattern, replacement, input);
```

Comment:
```suggestion
regexp.Replace(pattern, replacement, input)
```

Reply:
fixed this in all places. It was prettier automatically adding it